web/src/main/java/com/bazaarvoice/emodb/web/resources/sor/DataStoreResource1.java

```diff
@@ -133,6 +137,31 @@ public boolean include(Table table) {
         };
     }
 
+    @GET
+    @Path ("_unpublishedevents")
+    @Timed (name = "bv.emodb.sor.DataStoreResource1.listUnpublishedDatabusEvents", absolute = true)
+    @Unbuffered
+    @ApiOperation (value = "Returns all the emo table events that are not published on the databus.",
+            notes = "Returns a Iterator of a Map.",
+            response = Table.class
+    )
+    public Object listUnpublishedDatabusEvents(final @QueryParam ("from") DateTimeParam fromInclusiveParam,
+                                            final @QueryParam ("to") DateTimeParam toExclusiveParam,
+                                            final @Authenticated Subject subject) {
+        // Default date range is past 30 days
+        DateTime fromInclusive = fromInclusiveParam != null ? fromInclusiveParam.get() : DateTime.now().minusDays(29);
+        DateTime toExclusive = toExclusiveParam != null ? toExclusiveParam.get() : DateTime.now().plusDays(1);
```

Comment:
In my testing I ran into an unexpected corner case when the day 29 days ago was before daylight savings and the local timezone was "America/Chicago".  Because `minusDays(29)` was applied using the local timezone the end result was an hour off.  Recommend using `DateTime.now(DateTimeZone.UTC)` before doing the date arithmetic on the previous two lines.